routes/accounts/weekly_summary.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+const express = require( 'express' )
+const router = express.Router()
+const moment = require( 'moment' )
+const sequelize = require( 'sequelize' )
+
+
+const numberOfTasksCreated = ( Item, $gte, user_id ) =>
+  Item.findAndCountAll({ where: { createdAt: { $gte }, user_id } })
+    .then( result => result.count )
+
+
+const numberOfTasksCompleted = ( sequelize, user_id, since ) =>
+  sequelize.query(
+    `SELECT COUNT(DISTINCT item_id) FROM "Audits" WHERE field_name = 'completed' AND old_value = 'false' AND new_value = 'true' AND user_id = ${user_id} AND "createdAt" > '${since}'`,
+    { type: sequelize.QueryTypes.SELECT }
+  )
+
+const numberOfTasksEdited = ( sequelize, user_id, since ) =>
+  sequelize.query(
+    `SELECT COUNT(DISTINCT item_id) FROM "Audits" WHERE field_name = 'title' OR field_name ='description' AND old_value != new_value AND user_id = ${user_id} AND "createdAt" > '${since}'`,
+    { type: sequelize.QueryTypes.SELECT }
+  )
+
+const totalNumberOfTasks = ( Item, user_id ) =>
+  Item.findAndCountAll({ where: { user_id : user_id } })
```

Comment:
You can use implicit object creation (which will probably make this short enough to be a one liner):
```js
Item.findAndCountAll({ where: { user_id } }).then( ({ count }) => count )
```